.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,20 @@
+name: Node CI
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Node.js 6
```

Comment:
Node 6? Current version of node is 14. Is there a reason to be using such an early version? I know the WAT is fairly old, but I doubt 14 would break it, and if it does, I'd like to know about it.

Reply:
That was the version specified in `travis`. There shouldn't be any problem moving it to `14`.